### PR TITLE
Refacor and new commands surely they work Clueless

### PR DESCRIPTION
**Pull request by mynamecheff**
hopefully commands still work LULE

### CODE REVIEW

File: commands/help.js [hunk 1]
```diff
@@ -2,55 +2,43 @@ const tools = require('../tools/tools.js');
...
+    }
+
+    if (!input[2]) {
+      return 'List of commands: https://hotbear.org/ - If you want help with a command, write: "bb help *command*"';
+    }
+
+    input = input.slice(2).join(' ');
+    input = await tools.Alias(input);
+
+    const command = await sql.query('SELECT * FROM Commands WHERE Name=?', [input]);
```
Comment (JoachimFlottorp) [inline commands/help.js:23]: Clueless

File: commands/8ball.js [hunk 2]
```diff
@@ -1,26 +1,21 @@
...
+    description: 'This command will answer whatever yes/no question you have. Example: "bb 8ball Is NymN soy"',
+    permission: 100,
+    category: 'Random command',
+    noBanphrase: true,
+    execute: async (channel, user, input, perm) => {
+        try {
+            if (module.exports.permission > perm) {
+                return;
+            }
+            return responses[Math.floor(Math.random() * responses.length)];
```
Comment (hotbear1110) [inline commands/8ball.js:15]:
```suggestion
            return responses[~~(Math.random() * responses.length)];
```

File: commands/accage.js [hunk 3]
```diff
@@ -1,41 +1,41 @@
...
-			}
+            if (input[2]) {
+                if (input[2].startsWith('@')) {
+                    input[2] = input[2].substring(1);
+                }
+                username = input[2];
+            }
 
-			let twitchdata = await got(`https://api.ivr.fi/v2/twitch/user?login=${username}`).json();
+            const { body: twitchdata } = await got(`https://api.ivr.fi/v2/twitch/user?login=${username}`);
```
Comment (JoachimFlottorp) [inline commands/accage.js:25]: "twitchdata" is a string here

File: commands/cat.js [hunk 4]
```diff
@@ -1,28 +1,32 @@
-const { got } = require('./../got');
+const https = require('https');
```
Comment (hotbear1110) [inline commands/cat.js]:
```suggestion
const { got } = require('./../got');
```

File: commands/cat.js [hunk 5]
```diff
@@ -1,28 +1,32 @@
...
+    ping: true,
+    description: 'This command will give you a link to a picture of a random cat',
+    permission: 100,
+    category: 'Random command',
+    execute: async (channel, user, input, perm) => {
+        try {
+            if (module.exports.permission > perm) {
+                return;
+            }
+            https.get('https://api.thecatapi.com/v1/images/search', (res) => {
```
Comment (hotbear1110) [inline commands/cat.js]:
```suggestion
            got('https://api.thecatapi.com/v1/images/search', (res) => {
```

File: commands/cookme.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
+const fetch = require('node-fetch');
```
Comment (hotbear1110) [inline commands/cookme.js:1]: Would rather just use got

File: commands/forsenreddit.js [hunk 7]
```diff
@@ -0,0 +1,29 @@
+const { got } = require('got');
```
Comment (hotbear1110) [inline commands/forsenreddit.js:1]: Pretty sure forsens reddit isn't safe...

File: commands/help.js [hunk 8]
```diff
@@ -2,55 +2,43 @@ const tools = require('../tools/tools.js');
...
+      return;
+    }
+
+    if (!input[2]) {
+      return 'List of commands: https://hotbear.org/ - If you want help with a command, write: "bb help *command*"';
+    }
+
+    input = input.slice(2).join(' ');
+    input = await tools.Alias(input);
+
```
Comment (hotbear1110) [inline commands/help.js]:
```suggestion

const realcommand = input[1];

```

File: commands/help.js (hunk 1 above)
Comment (hotbear1110) [inline commands/help.js]:
```suggestion
    const command = await sql.query('SELECT * FROM Commands WHERE Name=?', [realcommand]);
```

Comment (JoachimFlottorp) [reply commands/help.js]: sql.Query

File: commands/randomtwitch.js [hunk 9]
```diff
@@ -0,0 +1,26 @@
+const { got } = require('got');
```
Comment (hotbear1110) [inline commands/randomtwitch.js:1]: Clueless kraken

File: commands/cat.js [hunk 10]
```diff
@@ -1,28 +0,0 @@
-const { got } = require('./../got');
```
Comment (hotbear1110) [inline commands/cat.js:1]: sus

File: commands/help.js [hunk 11]
```diff
@@ -1,56 +0,0 @@
-const tools = require('../tools/tools.js');
```
Comment (hotbear1110) [inline commands/help.js:1]: sus

**Review by hotbear1110 — CHANGES_REQUESTED** [review]
peepoTalk

**Comment by mynamecheff** [comment]
fixed?

**Review by hotbear1110 — APPROVED** [review]
Fixed the few issues I had with it

**Pull request merged**